scripts/modules/helpers.py

```diff
@@ -82,6 +83,75 @@ def load_images(urls, cache_dir):
 DEFAULT_HEALTHCHECK_RETRIES = 12
 
 
+def _print_done(service_name):
+    # 22 chars to ...
+    done = '\033[32mdone'
+    num_spaces = 21 - len(service_name)
+    s = ' '
+    dots = '...'
+    for _ in range(num_spaces):
+        s = s + ' '
+    print("{service}{spaces}{dots} {done}".format(service=service_name, spaces=s, dots=dots, done=done))
+
+
+def try_to_set_slowlog():
+    # This is a bit tricky to follow. What we're doing here is forking a "manager"
+    # process that will occasionally attempt to configure the slow log. After it
+    # detects that the slowlog has successfully been configured, it declares victory
+    # and terminates itself.
+    manager_process = multiprocessing.Process(target=_set_slowlog_json)
+    manager_process.start()
+
+
+def _set_slowlog_json():
+    this_try = 0
+    tries = 30
+    while this_try <= tries:
+        this_try += 1
+        time.sleep(3)
+        completed_process = subprocess.run([
+            "curl",
+            "--fail-early",
+            "-f",
+            "--connect-timeout",
+            "1",
+            "-u",
+            "admin:changeme",
```

Comment:
Could this password overwritten by anybody else? Or is it always the same?

Reply:
ouch!! I did not see it, yes it can be configurable

Reply:
Could somebody point me to how this might be done? It appears to me that the list of users [is pulled from here](https://github.com/elastic/apm-integration-testing/blob/master/docker/elasticsearch/users).

Searching for an option to overwrite this password doesn't yield anything obvious:

```
13:51 $ ./scripts/compose.py list-options|egrep pass
--apm-server-elasticsearch-password
--filebeat-elasticsearch-password
--heartbeat-elasticsearch-password
--metricbeat-elasticsearch-password
--packetbeat-elasticsearch-password
```

How does one overwrite the `admin` password on start, except to modify the `users` file in-lace?

Reply:
`--apm-server-elasticsearch-password` will be enough, all the reset should be the same if the use the main Elasticsearch, if not, it does not matter.

Reply:
I'm still a little confused. Are you saying that if a user passes in the `--apm-server-elasticsearch-password` flag that Elasticsearch won't start with `admin:changeme` as specified in the `users` file?

Reply:
if you pass `--apm-server-elasticsearch-password` you are probably not using the Elasticsearch instance in the `docker-compose` you ara using Elastic Cloud or ECK, so this admin user does not exists